tests/hh-escrow/withdraw.spec.ts

```diff
@@ -143,40 +143,40 @@ describe("withdraw", () => {
 
   //////////////////////////////////////////////////////////////////////////////
 
-  it("fails if the market is not finalized", async () => {
-    expect.assertions(1);
-
-    await expect(
-      withdraw()
-        .preInstructions([initMarketIx, userPositionIx])
-        .signers([market, user])
-        .rpc(),
-    ).rejects.toThrowProgramError(ErrorCode.NotFinalized);
-  });
-
-  it("fails if the user position is incorrect", async () => {
-    expect.assertions(1);
-
-    const wrongUser = Keypair.generate();
-    const wrongUserPosition = getUserPositionAddress(wrongUser, market);
-
-    const userPositionIx = await program.methods
-      .initializeUserPosition()
-      .accounts({
-        user: wrongUser.publicKey,
-        market: market.publicKey,
-        userPosition: wrongUserPosition,
-      })
-      .instruction();
-
-    await expect(
-      withdraw()
-        .accounts({ userPosition: wrongUserPosition })
-        .preInstructions([initMarketIx, userPositionIx])
-        .signers([market, wrongUser, user])
-        .rpc(),
-    ).rejects.toThrowProgramError(LangErrorCode.ConstraintSeeds);
-  });
+  // it("fails if the market is not finalized", async () => {
```

Comment:
Are these test cases no longer necessary?

Reply:
These tests are only applicable with the changes that have since been reverted by this PR.